source/javascripts/try_ruby.js.rb

```diff
@@ -342,29 +363,38 @@ def on_editor_change
   # End of playground methods
 
   def get_code_fragment(str)
-    # Let jQuery find the first code fragment in tryruby-content
-    code = Element.find('#tryruby-content code').html.strip
-    return code.gsub("&lt;", "<").gsub("&gt;", ">").gsub("&amp;", "&")
+    $document
+      .at_css('#tryruby-content code')
+      &.inner_html
+      .to_s
+      .strip
+      .gsub("&lt;", "<")
+      .gsub("&gt;", ">")
+      .gsub("&amp;", "&")
+  end
+
+  def title_element
+    $document.at_css("#tryruby-title")
   end
 
   def update_screen(item)
     return if !item
+
     @updating = true
-    Element.find('#tryruby-title').html   = item.title
-    Element.find('#tryruby-content').html = item.text
-    Element.find('#tryruby-answer').html  = item.answer
```

Comment:
This is a removal of a feature... I think it should be added back

Reply:
My bad, I thought this was superfluous since the `#tryruby-answer` div always has `display: none`. Is that for debugging purposes, perhaps?

It’s now back!